test/test_playback.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import, unicode_literals
+
+from mozlog.structuredlog import set_default_logger, StructuredLogger
+set_default_logger(StructuredLogger('test_playback'))
+
+from raptor.playback import get_playback, Mitmproxy  # noqa: E402
```

Comment:
Not an issue, but just pointing out that in mozilla-central E402 is disabled globally, so we'll have to remember to remove all of these. Guess we should just do that in our own flake8 config here. Oh well.

Reply:
Cool thanks